api/ranked1v1.py

```diff
@@ -46,20 +46,82 @@ def ranked1v1():
         where += " l.login LIKE %(player)s"
         args['player'] = '%' + player + '%'
 
-    return fetch_data(Ranked1v1Schema(), TABLE, SELECT_EXPRESSIONS, MAX_PAGE_SIZE, request, sort='-rating',
+    select = append_select_expression()
+
+    return fetch_data(Ranked1v1Schema(), TABLE1V1, select, MAX_PAGE_SIZE, request, sort='-rating',
                       args=args, where=where)
 
+@app.route('/rating/<string:rating_type>')
+def rating_type(rating_type):
+    if request.values.get('sort'):
+        raise InvalidUsage('Sorting is not supported for ranked1v1')
+
+    page = int(request.values.get('page[number]', 1))
+    page_size = int(request.values.get('page[size]', MAX_PAGE_SIZE))
+    player = request.args.get('filter[player]')
```

Comment:
(did we discuss this in slack?) There should not be a filter for player. For single players, there is a separate route